pkg/users/rename-group-dialog.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import { superuser } from "superuser";
+import React from 'react';
+import { Form, FormGroup, TextInput } from '@patternfly/react-core';
+
+import { apply_modal_dialog, show_modal_dialog } from "cockpit-components-dialog.jsx";
+
+const _ = cockpit.gettext;
+
+function RenameGroupDialogBody({ state, change }) {
+    const { name } = state;
+
+    return (
+        <Form isHorizontal onSubmit={apply_modal_dialog}>
+            <FormGroup fieldId="group-name" hasNoPaddingTop={!superuser.allowed} label={_("New name")}>
```

Comment:
Would you ever be able to even see this dialog without admin privs?

Reply:
Nope, redundant check. Removed.